DemoApp/appcenter-post-clone.sh

```diff
@@ -12,3 +13,14 @@ echo 'Add private cocoapods repository'
 pod repo add $REPO_NAME https://$USER_ACCOUNT:$ACCESS_TOKEN@$PRIVATE_REPO_BASE_URL/$REPO_NAME
 pod repo update
 cd ios && pod install
+
+# Remove unnessesary NDKs, this clean is needed to avoid failures on AppCenter CI which requires only NDK21. 
```

Comment:
```
Remove all NDKs except NDK 21.+ to avoid failures on App Center CI with error:
> No toolchains found in the NDK toolchains folder for ABI with prefix: arm-linux-androideabi
```